web/src/lib/components/memory-page/memory-viewer.svelte

```diff
@@ -125,15 +132,9 @@
 			const canAdvance = toNextMemory();
 			if (!canAdvance) {
 				autoPlay = false;
```

Comment:
Though, in reference to my other comment, this does not use `toggleAutoPlay` and so the condition may actually be required.